.github/workflows/pr-builder.yml

```diff
@@ -190,19 +190,14 @@ jobs:
         run:  pnpm build:modules
 
       - name: 🃏 Run Jest & Collect Coverage
-        id: run-jest-test-and-coverage
-        # TODO: Update the command to "test:unit:coverage" once the coverage command related issues
-        # are sorted out
+        id: run-jest-tests
```

Comment:
is this line needed

Reply:
Added the id here as well for consistency.